src/cdk/tree/tree.ts

```diff
@@ -307,6 +330,16 @@ export class CdkTree<T, K = T> implements AfterContentChecked, CollectionViewer,
       CdkTreeNode.mostRecentTreeNode.data = nodeData;
     }
   }
+
+  /** Level accessor, used for compatibility between the old Tree and new Tree */
+  private getLevelAccessor() {
+    return this.treeControl?.getLevel ?? this.levelAccessor;
+  }
+
+  /** Children accessor, used for compatibility between the old Tree and new Tree */
+  private getChildrenAccessor() {
+    return this.treeControl?.getChildren ?? this.childrenAccessor;
+  }
```

Comment:
Is your intent to add unit tests once more of the pieces have landed in the development branch?

Reply:
Yep, that's the intent here.